lib/active_model_serializers/railtie.rb

```diff
@@ -13,11 +13,12 @@ class Railtie < Rails::Railtie
       ActiveModel::Serializer.serializers_cache.clear
     end
 
-    initializer 'active_model_serializers.action_controller' do
-      ActiveSupport.on_load(:action_controller) do
-        include(::ActionController::Serialization)
-      end
-    end
+    # We do not want this mix
+    # initializer 'active_model_serializers.action_controller' do
```

Comment:
How about adding option to disable insert mixin to  `ActiveModelSerializers.config`  instead? Adding option makes the difference of the default behavior between ours and the original's one smaller.

Reply:
Yes I was thinking about it but I was unsure how to test this and/or how to set this option before the railtie is ran, I think it might be possible to do so in application.rb 🤔

Reply:
I've confirmed this works properly as expected, so thanks for suggesting this 👍

Reply:
👍👍👍 Would you share the snippets to disable this option on here? It would be useful for other developers.

Reply:
Yeah sure I've added the example, or here in `application.rb` we can have:

```ruby
    config.before_initialize do
      ActiveModelSerializers.config.mixes_action_controller = false
    end
```